code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -58,9 +58,11 @@
 	var/temperature = FIRE_MINIMUM_TEMPERATURE_TO_EXIST
 	var/just_spawned = TRUE
 	var/bypassing = FALSE
+	var/datum/looping_sound/atmos_fire/soundloop
```

Comment:
make this a `static` var and add/remove `src` to its `output_atoms` instead of instantiating/qdelling a new loop for every hotspot.